optaplanner-core/src/main/java/org/optaplanner/core/config/score/director/ScoreDirectorFactoryConfig.java

```diff
@@ -122,6 +122,10 @@
     protected List<File> scoreDrlFileList = null;
     @XStreamConverter(KeyAsElementMapConverter.class)
     protected Map<String, String> kieBaseConfigurationProperties = null;
```

Comment:
I wonder... shouldn't this be deprecated too, along with the `scoreDrlFileList`? What's it for?

Reply:
I'd argue no, because they can also be used in combination with scoreDrl.
For example to turn on switch that hows-it-called propagation from "ALLOWED" to "ALWAYS"